api/src/main/resources/workbench-api.yaml

```diff
@@ -5345,6 +5385,15 @@ definitions:
         format: int64
         description: 'Timestamp when the user first changed from Unregistered to Registered
           data access level. May be null if the user has never done so.'
+  LinkedAccountInfo:
+    description: 'User valid account linking information. Getting this response means the account is
+    validated. For now, we just need eRA Commons username, moving forward, we may add login.gov,
+    IAL status when needed. Getting '
+    type: object
+    properties:
+      eraCommonsUserName:
```

Comment:
Do we actually need to return this to the client? What is the use case?

Reply:
I am thinking about UI can use those information to do different validations.  Like we returned eRA, login.gov, IAL2.
UI will get enough information on what to check in different phases.